### PR TITLE
Move getColumnMapFromSql() to SystemDatabase

### PR DESCRIPTION
**Pull request by kamil-tekiela**

### CODE REVIEW

File: libraries/classes/SystemColumn.php [hunk 1]
```diff
@@ -0,0 +1,22 @@
+<?php
+
+declare(strict_types=1);
+
+namespace PhpMyAdmin;
+
+class SystemColumn
```
Comment (williamdes) [inline libraries/classes/SystemColumn.php:7]: It could have psalm sealed or readonly phpdoc tags that other classes have?

**Review by williamdes — APPROVED** [review]

File: libraries/classes/SystemColumn.php (hunk 1 above)
Comment (kamil-tekiela) [reply libraries/classes/SystemColumn.php:10]: Good idea. I added `@psalm-immutable`

**Review by MauricioFauth — APPROVED** [review]

File: libraries/classes/SystemColumn.php [hunk 2]
```diff
@@ -0,0 +1,25 @@
...
+
+namespace PhpMyAdmin;
+
+/**
+ * @psalm-immutable
+ */
+class SystemColumn
+{
+    /** @var string */
+    public $tableName;
```
Comment (MauricioFauth) [inline libraries/classes/SystemColumn.php:13]:
```suggestion
    public readonly string $tableName;
```

**Review by MauricioFauth — CHANGES_REQUESTED** [review]

File: libraries/classes/SystemColumn.php (hunk 2 above)
Comment (kamil-tekiela) [reply libraries/classes/SystemColumn.php]: Yeah, this is completely unnecessary now 😄

**Review by MauricioFauth — APPROVED** [review]

**Pull request merged**